templates/contract-deploy/deploy_parameters.json

```diff
@@ -9,7 +9,7 @@
     "multiplierGas": "",
     "pendingStateTimeout": 604799,
     "polTokenAddress":"",
-    "salt": "0x0000000000000000000000000000000000000000000000000000000000000001",
+    "salt": "0x6e0d215341c328184a6f634169e29e223cb74efb907e94de3a2773f1c805cf08",
```

Comment:
Should we use this new `salt` value?

Reply:
No, I should revert this back to 0x0. This value is of course valid, but it would cause a different address to be created during deployment which might make some documentation and scripts harder to follow.